assets/js/map.js

```diff
@@ -119,10 +119,6 @@ const MapBase = {
       zoomControl: false,
       crs: L.CRS.Simple,
       layers: [mapLayers[this.themeOverride || Settings.baseLayer]],
-      zoomSnap: 0,
-      zoomDelta: 0.5,
-      wheelPxPerZoomLevel: 70,
-      wheelDebounceTime: 150,
```

Comment:
This values were here on purpose to unify zoom on mouse scroll.
`zoomSnap: 0` allows on any value zoom on mobile devices (useful on small screens),
`zoomDelta: 0.5` makes 10 usable zoom levels, if you remove that value it sometimes skip some map zoom levels,
 `wheelDebounceTime` i had to increase that value to 150, otherwise map was very laggy on fast zoom.
If that's not a crucial thing to fix the issue, i would leave it as it is.

Reply:
@MichalD96 Good to see you back!
It's temporary. Options for these parameters have been added in the next PR, using the default values from the Leaflet library as the default for this map. Users can change settings, and it is no longer fixed to a specific value. Perhaps the help text for options could be improved further. Please preview it.

Reply:
Alright, but let's make this values as default.
I checked #1275 PR and there is only 5 usable zoom levels, when supposed to be 10

Reply:
Got it done.